bin/utils.cjs

```diff
@@ -50,6 +50,14 @@ async function getPackageJson() {
   }
 }
 
+async function setGithubAccessToken() {
+  const prompt = new Password({ message: "GitHub PAT token" });
```

Comment:
Can we make this step optional? Generally speaking, our developers will likely have their PAT already set in `~/.npmrc` via calling `npm login --scope=@harvard-lxp --auth-type=legacy --registry=https://npm.pkg.github.com`, so there shouldn't be a need to have it also set in the local `.npmrc` file.

Reply:
Sure